### ISSUE TITLE
resetting sim does not reset all descriptions

### ISSUE THREAD

**Issue by zepumph** [phetsims/gravity-force-lab-basics#116]
In particular the aria-valuetext for the position sliders. On sim start it looks like "Left side of the track, close to mass 2." And on restart it could look like "Farther away, close to mass 2." 

This is because on reset we move the slider from closer to the center back to the starting point, which is "farther away" than where it was. We need to go through much of this logic and make the the reset case is being handled correctly.

@terracoda for interview tomorrow, can we tell the interviewer to just refresh the page instead of using the reset all button?

**Comment by terracoda** [comment]
A refresh is a reasonable work around for interviews.

**Comment by terracoda** [comment]
@zepumph, I cannot reproduce this issue with Safari & VoiceOver on MacOS 10.13.6 (High Sierra). Hmm, I thought I upgraded to Mohave. Apparently, not so - just a security update.

**Comment by zepumph** [comment]
The original bug reported in this issue wasn't actually changing the screen reader output, one issue is in the PDOM. On reset, the aria-valuetexts for position sliders have outdated values, but on focus they are updated again to be correct. It is this discrepancy that caused me to make this issue more as a way to audit the behavior of the reset all button, and to make sure it was always working as expected.

**Comment by terracoda** [comment]
@zepumph, when you put it that way, yes, in the A11y View I think I have seen this. I'll try to verify.

**Comment by terracoda** [comment]
@zepumph, yes this is only an issue for the `aria-valuetext` for the Mass Sphere position sliders. Everything else seems to reset fine.

I suppose if the user read through the PDOM after a reset they might get stale information, so I agree this is an issue.

Here's a screen shot of the A11y View just where the `aria-valuetext` on the position sliders is stale.
<img width="1124" alt="Screen Shot 2019-04-15 at 11 08 10 AM" src="https://user-images.githubusercontent.com/1902116/56120882-5eeb5380-5f6f-11e9-9894-5ff29bec183b.png">

**Comment by zepumph** [comment]
@terracoda, I also only see this in the aria-valuetext. This was fixed for the mass controls in https://github.com/phetsims/gravity-force-lab-basics/issues/125, and will be handled for the position sliders in https://github.com/phetsims/gravity-force-lab-basics/issues/124. Closing this issue.
